extensions/wikia/PortableInfoboxBuilder/services/PortableInfoboxBuilderService.class.php

```diff
@@ -245,12 +246,21 @@ protected function importNodeToDom( $domDocument, $childNodeDom ) {
 	}
 
 	/**
+	 * @param $collapse
 	 * @return DOMElement
 	 */
-	protected function createGroupDom() {
-		return dom_import_simplexml(
-			new SimpleXMLElement( '<' . \Wikia\PortableInfoboxBuilder\Nodes\NodeGroup::XML_TAG_NAME . '/>' )
-		);
+	protected function createGroupDom($childHeader, $childNodeDom) {
```

Comment:
hmm, what do you think about
```
protected function createGroupDom($headerNodeDom, $collapsible) { ... }
```
invoked in 
https://github.com/Wikia/app/pull/9819/files#diff-f997a9b22af5a0ef703dfc816b8678ebR202 
and
https://github.com/Wikia/app/pull/9819/files#diff-f997a9b22af5a0ef703dfc816b8678ebR216
like
```
$currentGroupDom = $this->createGroupDom( $childNodeDom, $this->isCollapsibleHeader($currentChildNode));
```
That would make the createGroupDom more explicit and ensure the share of responsibilities.